Resources/Prototypes/Magic/gate_spells.yml

```diff
@@ -0,0 +1,11 @@
+- type: instantAction
+  id: Gate
+  name: action-name-spell-gate
+  description: action-description-spell-gate
+  speech: action-speech-spell-gate
+  useDelay: 60
+  itemIconStyle: BigAction
+  icon:
+    sprite: Effects/portal.rsi
+    state: portal-green
+  serverEvent: !type:GateSpellEvent
```

Comment:
linter is failing because actions where changed recently to not suppor speech, take a look at how the old spells got ported here https://github.com/space-wizards/space-station-14/pull/15055